cpp/src/parquet/file_deserialize_test.cc

```diff
@@ -879,6 +879,38 @@ TEST_F(TestPageSerde, DataPageV2CrcCheckNonExistent) {
                          /* write_data_page_v2 */ true);
 }
 
+TEST_F(TestPageSerde, BadCompressedPageSize) {
+  auto codec_types = GetSupportedCodecTypes();
+  const int32_t num_rows = 32;  // dummy value
+  data_page_header_.num_values = num_rows;
+  std::vector<uint8_t> faux_data;
+  test::random_bytes(1024, 0, &faux_data);
+  for (auto codec_type : codec_types) {
```

Comment:
In my local pc, the brotli will "success" decompression.

I've read the code of `BrotliCodec`, but it has a `output_size` and it's used. I didn't dive into it.

Reply:
Ah I've make a mistake and confuse the size.